### ISSUE TITLE
cxcy_to_gcxgcy function return nan values

### ISSUE THREAD

**Issue by Eloneinstein** [sgrvinod/a-PyTorch-Tutorial-to-Object-Detection#54]
I traced back where the nan is coming from the loss function. It leads to `cxcy_to_gcxgcy` function.
Division is causing nan value it seems.  please provide any solution for this.

**Comment by Tathagatd96** [comment]
I am facing the same issue. @Eloneinstein , did u find any solution?

**Comment by liebkne** [comment]
@Eloneinstein @Tathagatd96 Have you fixed this issue?

**Comment by TommyHuang821** [comment]
In my observation, it may happen when mis-order for bbox.
BBOX usually places with order [left-top(x),left-top(y), right-down(x), right-down(y)], but nan will occur with [right-down(x), right-down(y), left-top(x),left-top(y)].

**Comment by m-evdokimov** [comment]
@TommyHuang821 that was my problem, thanks!

**Comment by destr4ct** [comment]
> In my observation, it may happen when mis-order for bbox. BBOX usually places with order [left-top(x),left-top(y), right-down(x), right-down(y)], but nan will occur with [right-down(x), right-down(y), left-top(x),left-top(y)].



> @TommyHuang821 that was my problem, thanks!
Same!

**Comment by JYeonKim** [comment]
@destr4ct  Hi! I have the same problem. How did you solve it? I'd really appreciate it if you told me. thank you!
